### DISCUSSION TITLE
3.14 is not "stable" Everything is broken

### DISCUSSION THREAD

**Discussion by hoshi411** [florisboard/florisboard#1753] [Q&A]
V. 0.3.14 (75)
Everything is broken.
Glide doesnt work
Suggestions dont work

How is this "stable"

I only got the theme working because i came to this page and searched how to use it.

Its NOT stable .  If i cant see suggestions and spelling and autocorrect and glide is broken, that is not stable.

Please put this back in beta until these issues are fixed.  I would not have upgraded .

Can we please make I capital?  I hate having to hit the caps lock every time I type "I" 

Oh and thank you for the best android keyboard there is.  Some of the new features are awesome. Glide actually works better than before. I love the new gestures for space bar and delete button!!!!

22/04/05 22:46
> After force quit and restarting florisboard, glide is now working . But without spelling and suggestions it is so hard to type.

**Comment by patrickgold** [comment]
If you look at the first paragraph of the README, you see the following:

![image](https://user-images.githubusercontent.com/19412843/161815138-5ce6b4c9-bcca-4c52-8b12-06a98322f006.png)

So, FlorisBoard as an entire project is in early beta, and I'd say that this is correct.

"Stable" releases with a `0` prefix just mean, that they are the most stable releases within the current overall state of this project, which isn't at all production-ready. And you cannot reasonably expect that an early-beta keyboard already has superb word suggestions, glide typing and everything else, that's just delusional. Things take time to implement correctly, that's the sad truth.

I agree that glide typing is very buggy but there are only so much things I can fix at once, and glide typing is horrifically difficult to fix due to the current code base. Word suggestions is the next pain topic but at least that is the only major goal for 0.4.0 (which will be started to work on after the 0.3.16 bug fix and polishing release), and this time I can learn from my various previous mistakes on word suggestions attempt 1, so hopefully I get it right this time.

Hope you understand my point of view.

**Comment by hoshi411** [comment]
Like I mentioned, this is my favorite android keyboard.  In my opinion the best android keyboard that there is.  The update was a shock to me, initially.  So I reacted as I did since there were more working features before the upgrade.  
I understand that the current official release is a beta and even then I of course accept this keyboard with any flaws I perceive. 
Please keep up the good work.  The devs of this keyboard are Heros / a hero. 

Looking forward to when suggestions work again.

Thank you.

**Reply by silverwings15** [reply]
Florisboard has been making huge strides in the past few months and is well on its way to achieve feature parity with Gboard/Swiftkey, a massive feat for a FOSS keyboard. i personally can't wait until I can use it to fully replace Swiftkey. until then, no point in criticizing Florisboard for not being the complete product yet imo. more constructive to create bug reports and make feature suggestions to help the project improve.

**Reply by hoshi411** [reply]
sigh.
